runner/src/server/utils/configSchema.ts

```diff
@@ -78,6 +78,29 @@ export const configSchema = Joi.object({
   safelist: Joi.array().items(Joi.string()),
   initialisedSessionTimeout: Joi.number(),
   initialisedSessionKey: Joi.string(),
+  initialisedSessionAlgorithm: Joi.string()
+    .allow(
+      "RS256",
+      "RS384",
+      "RS512",
+      "PS256",
+      "PS384",
+      "PS512",
+      "ES256",
+      "ES384",
+      "ES512",
+      "EdDSA",
+      "RS256",
+      "RS384",
+      "RS512",
+      "PS256",
+      "PS384",
+      "PS512",
+      "HS256",
+      "HS384",
+      "HS512"
+    )
+    .default("HS512"),
```

Comment:
Just out of curiosity, are any of these algorithms ones that we should discourage the use of (by not including)?

Reply:
Anything under 128 bits but as I understand it HS-1 still takes considerable amount of time/hashing power to crack. Before which you will notice increased traffic or the stored session will expire. 

The only alg I omit is “none”